### PR TITLE
Adds test for update_df from utils

### PR DESCRIPTION
**Pull request by Kadam-Tushar**
- Adds a test for [update_df ](https://github.com/cleanlab/cleanvision/blob/main/src/cleanvision/utils/utils.py#L100) function from utils

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -15,3 +18,75 @@ def test_get_filepaths(self, tmp_path):
...
+        only_new_col_names = ["new" + str(x) for x in range(num_only_new_cols)]
+        new_df = pd.DataFrame(
+            np.random.rand(num_rows, num_only_new_cols), columns=only_new_col_names
+        )
+
+        # adding common cols
+        common_col_names = ["com" + str(x) for x in range(num_common_cols)]
+        old_df[common_col_names] = np.random.rand(num_rows, num_common_cols)
+        new_df[common_col_names] = np.random.rand(num_rows, num_common_cols)
+        print("old df ")
```
Comment (sanjanag) [inline tests/test_utils.py:53]: Remove print statements

File: tests/test_utils.py [hunk 2]
```diff
@@ -15,3 +18,75 @@ def test_get_filepaths(self, tmp_path):
         filepaths = get_filepaths(tmp_path)
         print(filepaths)
         assert len(filepaths) == len(extensions) * num_names
+
+    @pytest.mark.parametrize(
```
Comment (sanjanag) [inline tests/test_utils.py:22]: `overwrite` can also be parametrized here

File: tests/test_utils.py [hunk 3]
```diff
@@ -15,3 +18,75 @@ def test_get_filepaths(self, tmp_path):
...
+        )
+        assert num_only_new_cols == 0 or updated_df[only_new_col_names].equals(
+            new_df[only_new_col_names]
+        )
+        assert num_common_cols == 0 or updated_df[common_col_names].equals(
+            new_df[common_col_names]
+        )
+
+        # testing overrite = False
+        updated_df = update_df(old_df, new_df, overwrite=False)
```
Comment (sanjanag) [inline tests/test_utils.py:77]: paremetrizing overwrite will reduce all these asserts here and for overlapping columns you can simply use an if conditions
```suggestion
    if overwrite:
        assert num_common_cols == 0 or updated_df[common_col_names].equals(
            new_df[common_col_names]
        )
    else:
        assert num_common_cols == 0 or updated_df[common_col_names].equals(
            old_df[common_col_names]
        )
        
        
        
```

**Comment by Kadam-Tushar** [comment]
Thank you for your suggestions. @sanjanag
Updated the PR!

**Review by Kadam-Tushar — COMMENTED** [review]
Refactored function as mentioned in the review.

**Review by sanjanag — APPROVED** [review]

**Pull request merged**